.github/workflows/commands.yml

```diff
@@ -105,22 +46,10 @@ jobs:
         reaction: "true"
         reaction-type: "eyes"
         allow-edits: "false"
-        permission-level: write
+        permission-level: read
     - name: Handle Command
       uses: actions/github-script@v4
```

Comment:
```suggestion
```

Reply:
Ah, thanks. I had fixed this but forgot to push the updated commit. Done.